include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -128,19 +128,16 @@ class alphabet_tuple_base :
                                 (1 * ... * alphabet_size<component_types>),
                                 void>; // no char type, because this is only semi_alphabet
 
-    //!\brief A meta::list The types of each component in the composite
-    using component_list = meta::list<component_types...>;
+    //!\brief A seqan3::type_list The types of each component in the composite
+    using component_list = seqan3::type_list<component_types...>;
 
     //!\brief Is set to `true` if the type is contained in the type list.
     template <typename type>
-    static constexpr bool is_component =
-        meta::in<component_list, type>::value;
+    static constexpr bool is_component = seqan3::list_traits::contains<type, component_list>;
 
     //!\brief Is set to `true` if the type is uniquely contained in the type list.
     template <typename type>
-    static constexpr bool is_unique_component =
-        is_component<type> &&
-        (meta::find_index<component_list, type>::value == meta::reverse_find_index<component_list, type>::value);
+    static constexpr bool is_unique_component = (seqan3::list_traits::count<type, component_list> == 1);
```

Comment:
Actually does not need parenthesis because it's not a concept, but they don't hurt either; actually easier to read with `()`

Reply:
No hard feeling :)